services/ui-src/src/components/fields/TextField.tsx

```diff
@@ -84,6 +85,8 @@ export const TextField = ({
 
   return (
     <Box sx={sxOverride} className={`${nestedChildClasses} ${labelClass}`}>
+      {/* SAR field sections */}
+      {heading && <Heading sx={sx.fieldHeading}>{heading}</Heading>}
```

Comment:
🤔 is it possible to add this kind of thing at a higher level so it doesn't have to be added to each field type that requires it? might be over-engineering

Reply:
yeah! I wasn't sure where to put this, so I just added it for now where it's being used, but I can cut a ticket to SPIKE it out 🤔